guides/nestor/ISPsetup.md

```diff
@@ -0,0 +1,308 @@
+
+![Selection9](https://i.imgur.com/LJgDvBJ.png)
+
+- `Then press control+X to save the config file on ubuntu and enter Y to safe the file` 
+
+![Selection9](https://i.imgur.com/D2IAI1b.png)
+
+- `Then press Enter to make sure the file name stay the same`
+
+![Selection9](https://i.imgur.com/aWGEO3P.png)
+
+- `Next step run command (ip link show) to see the network interfaces like bellow`
+
+![Selection9](https://i.imgur.com/2r0ghbi.png)
+
+- `You will need to add another interface to the server`
+- `Run command on the terminal (sudo nano /etc/network/interfaces) and enter your admin password`
+
+![Selection9](https://i.imgur.com/EmMewtr.png)
+
+- `Then you will see a text file like bellow` 
+
+![Selection9](https://i.imgur.com/YzAyL1s.png)
+
+- `Then add the new network adapter like bellow`
+
+![Selection9](https://i.imgur.com/cU2izxY.png)
+
+- `Then press (control X) to exit and (Y) to save file then enter to leave the file with the same name do not change the file name`
+
+![Selection9](https://i.imgur.com/gCgEyWU.png)
+
+- `run (sudo reboot) on the terminal for the interfaces to take effect` 
+- `Then run router on the terminal shout look like bellow`
+
+![Selection9](https://i.imgur.com/wCEIwLd.png)
+
+- `Then run the next line on terminal`
+- `(sudo route add –net 172.0.0.0 netmask 255.0.0.0 gw 172.31.1.3 dev enp0s8)`
+
+![Selection9](https://i.imgur.com/AveqQd6.png)
+
+- `Then run route again it should look like bellow`
+
+![Selection9](https://i.imgur.com/I9QDTpD.png)
+
+- `Then run (sudo services networking restart) to restard the network services again nothing will display after this command and password is enter`
+
+![Selection9](https://i.imgur.com/hR7cdtQ.png)
+
+- `Then to verify everything is working fine run (ifconfig) it should look like bellow`
+
+![Selection9](https://i.imgur.com/b4lVh9W.png)
+
+- `Then run this tree line on terminal like the picture bellow`
+- `sudo apt-get install -y iptables-persistent`
+
+![Selection9](https://i.imgur.com/xncEGuM.png)
+
+- `Then on the next two question select yes like bellow`
+
+![Selection9](https://i.imgur.com/yjjPrcx.png)
+![Selection9](https://i.imgur.com/Fjd9QBr.png)
+
+- `Run next line bellow and password if ask it will not display anything after you run the line`
+- `sudo iptables -t nat -A POSTROUTING -o enp0s3 -j MASQUERADE`
+
+![Selection9](https://i.imgur.com/jJrCz0Z.png)
+
+- `Run next line bellow and password if as and it will look like the picture bellows`
+- `sudo netfilter-persistent save`
+
+![Selection9](https://i.imgur.com/JTUsR1o.png)
+![Selection9](https://i.imgur.com/JamB7Pu.png)
+
+- `Run next line bellow and password if as and it will look like the picture bellows`
+- `sudo netfilter-persistent reload`
+
+![Imgur](https://i.imgur.com/SmgzpkB.png)
+![Imgur](https://i.imgur.com/njlW4bE.png)
+
+- `After all this reboot Virtual Machine and move to next step pfsense setup link (comming soon)`
```

Comment:
Your guide needs more verification blocks. When someone does a command on your guide they should know they did it correct. This will help troubleshooting since they will know what went wrong.

Reply:
what does this mean can you send me a example??